style.css

```diff
@@ -15,15 +15,15 @@ body {
   height: 400px;
   background-repeat: no-repeat;
   margin-right: 0;
-  list-style: none
+  list-style: none;
 }
 
 .container {
```

Comment:
- [ ] The icons on the headline section have too much margin, please stick to the design as much as possible

![too much margin here](https://user-images.githubusercontent.com/25227022/165838653-90bc3d8e-bb49-45ae-b4bf-2f4067687e16.png)